overlord/snapstate/backend_test.go

```diff
@@ -1266,7 +1266,24 @@ func (f *fakeSnappyBackend) SetupSnapSaveData(info *snap.Info, _ snap.Device, me
 	return f.maybeErrForLastOp()
 }
 
-func (f *fakeSnappyBackend) LinkSnap(info *snap.Info, dev snap.Device, linkCtx backend.LinkContext, tm timings.Measurer) (rebootInfo boot.RebootInfo, err error) {
+func (f *fakeSnappyBackend) MaybePrepareReboot(
+	info *snap.Info, dev snap.Device, isUndo bool) (boot.RebootInfo, error) {
```

Comment:
Might be worth capturing `isUndo` on the op to validate that it is set correctly?

Reply:
Good point, done